client/src/components/_common/Form/FileInputDropZone.js

```diff
@@ -88,23 +68,31 @@ function FileInputDropZone({
       {showFileList && (
         <div className="attachment-view">
           <div className="attachment-list">
-            {files.map((f, i) => (
-              <div className="attachment-block" key={[f.name, i].toString()}>
-                <span className="d-inline-block text-truncate">{f.name}</span>
-                <Button
-                  color="link"
-                  className="attachment-remove"
-                  onClick={() => {
-                    setNumberRejectedFiles(0);
-                    removeFile(i);
-                  }}
-                >
-                  Remove
-                </Button>
-              </div>
-            ))}
+            {rejectedFiles &&
+              rejectedFiles.map((f, i) => (
+                <div className="attachment-block" key={[f.name, i].toString()}>
+                  <span className="d-inline-block text-truncate">{f.name}</span>
+                  <InlineMessage type="error">
+                    Exceeds File Size Limit
+                  </InlineMessage>
```

Comment:
This matches design 👍 , but had me wondering. @tacc-wbomar Should there also be an option to remove rejected files? I guess there's no need technically since you can still upload valid files, but as a user you might expect that you could remove a rejected file.

Reply:
Probably. I expected to be able to, then realized the same thing you did. I'll ask. But this ticket is to improve error messages, which has been done; so I will not hold up my approval on it.

Reply:
[Asked.](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1621535170021500)